spec/scrabble_spec.rb

```diff
@@ -0,0 +1,83 @@
+require "./lib/scrabble"
+require "spec_helper"
+
+describe "Letters & Scores" do
+  # [
+  #   ["A", 1],
+  #   ["E", 1],
+  #   ["I", 1],
+  #   ["O", 1],
+  #   ["U", 1],
+  #   ["L", 1],
+  #   ["N", 1],
+  #   ["R", 1],
+  #   ["S", 1],
+  #   ["T", 1],
+  #   ["D", 2],
+  #   ["G", 2],
+  #   ["B", 3],
+  #   ["C", 3],
+  #   ["M", 3],
+  #   ["P", 3],
+  #   ["F", 4],
+  #   ["H", 4],
+  #   ["V", 4],
+  #   ["W", 4],
+  #   ["Y", 4],
+  #   ["K", 5],
+  #   ["J", 8],
+  #   ["X", 8],
+  #   ["Q", 10],
+  #   ["Z", 10]
+  # ]#.each do |letter, letter_score|
+
+  it "responds with the scores for the letters" do
+    ("A".."Z").each do |letter|
+      score_value = Scrabble::Scrabble.score(letter)
+      lookup_value = Scrabble::SCOREBOARD[letter]
+
+      expect(score_value).to eq(lookup_value)
+    end
+  end
+
+  it "splits the words into an array of letters" do
+    expect(Scrabble::Scrabble.letters_in_word("DOG")).to eq(["D","O","G"])
+  end
+
+  it "gets the score for each letter in word" do
+    expect(Scrabble::Scrabble.score("dog")).to eq(5)
```

Comment:
One way of being really clear about this test is to use variables like so:
```
d = Scrabble::SCOREBOARD['d']
o = Scrabble::SCOREBOARD['o']
g = Scrabble::SCOREBOARD['g']
expect(Scrabble::Scrabble.score('dog')).to eq(d + o + g)
```
This way, if you have to change a score on one of the letters, you won't have to update the test.